webapp/channels/src/components/widgets/inputs/users_emails_input.tsx

```diff
@@ -85,13 +87,14 @@ export default class UsersEmailsInput extends React.PureComponent<Props, State>
         loadingMessage: messages.loadingDefault,
         showError: false,
     };
-    private selectRef: RefObject<AsyncCreatable<UserProfile | EmailInvite> & { handleInputChange: (newValue: string, actionMeta: InputActionMeta | { action: 'custom' }) => string }>;
+    private selectRef: RefObject<SelectInstance<UserProfile | EmailInvite, boolean, GroupBase<UserProfile | EmailInvite>> & { onInputChange: (newValue: string, actionMeta: InputActionMeta | { action: 'custom' }) => string }>;
```

Comment:
Related to my previous comment, this `& {onInputChange: ...}` shouldn't be needed either as long as we pass a valid `InputActionMeta` object for this